services/app-api/forms/wp.json

```diff
@@ -914,7 +913,7 @@
             "deleteModalConfirmButtonText": "Yes, delete objective",
             "deleteModalWarning": "You will lose all information entered for this objective. Objective will not be deleted from archived Work Plans and Semi-Annual Reports, but it will not be available in future Work Plans and Semi-Annual Reports.",
             "enterEntityDetailsButtonText": "Edit",
-            "dashboardTitle": "Objective total count: 2",
+            "dashboardTitle": "Objective total count",
```

Comment:
Do we need a `countEntitiesInTitle: true` flag here?

Reply:
Good catch. Never knew about this field. Added!